Dockerfile.compute-node

```diff
@@ -650,6 +667,10 @@ COPY --from=pg-pgx-ulid-build /usr/local/pgsql/ /usr/local/pgsql/
 COPY --from=rdkit-pg-build /usr/local/pgsql/ /usr/local/pgsql/
 COPY --from=pg-uuidv7-pg-build /usr/local/pgsql/ /usr/local/pgsql/
 COPY --from=pg-roaringbitmap-pg-build /usr/local/pgsql/ /usr/local/pgsql/
+# For transition period we keep private extenstions in compute-node image as well
+# Private extenstions
+COPY --from=pg-anon-pg-build /usr/local/pgsql/ /usr/local/pgsql/anon/
+
```

Comment:
I want to use this pg-anon extension example to test downloading of the extension that is **not** present in compute image.

So, if I read it correctly this code should be removed

Reply:
For the transition period, we use a compute-node image as a compute-node itself and as a container for extensions that we upload to S3 (later it'll be a separate image). I can change the path to these files (to something like `/anon`) so that they definitely won't be loaded automatically

Reply:
> Is it difficult to implement separate image?
> We definitely need transition logic for already supported extensions, but maybe we can avoid it for the new one?

Replaced with `extenstions-vXX` image, which we're going to use eventually for all extensions.